pycode/memilio-simulation/CMakeLists.txt

```diff
@@ -105,3 +121,72 @@ add_pymio_module(_simulation_omseirs4
     LINKED_LIBRARIES memilio ode_mseirs4
     SOURCES memilio/simulation/bindings/models/omseirs4.cpp
 )
+
+set(MEMILIO_SIMULATION_LIBS
+    memilio
+    abm
+    ode_sir
+    ode_seir
+    ode_secir
+    ode_seirdb
+    ode_secirvvs
+    sde_sir
+    sde_sirs
+    ode_mseirs4
+)
+
+foreach(target_name IN LISTS MEMILIO_SIMULATION_LIBS)
+    if(TARGET ${target_name})
+        set_target_properties(${target_name} PROPERTIES
+            INSTALL_RPATH "$ORIGIN"
+        )
+    endif()
+endforeach()
+
+install(TARGETS ${MEMILIO_SIMULATION_LIBS}
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+    RUNTIME DESTINATION lib
+)
```

Comment:
Why are these not already set in the add_pymio_module function? That way, we don't have to maintain two lists

Reply:
Yes, that might not be the best solution. However, the `add_pymio_module` only handles the pybind modules. This list `MEMILIO_SIMULATION_LIBS` containts the c++ libs from the included cpp project. 
TO install them (once!) in the wheel structure, they need an own install block.
If we would include this in `add_pymio_module`, each module would have to install these as well (leading to duplicates).
Therefore, i decided to use a seperate list.